dotCMS/src/integration-test/java/com/dotcms/datagen/ContainerAsFileDataGen.java

```diff
@@ -16,20 +16,33 @@
 import com.google.common.collect.ImmutableSet;
 import com.liferay.util.FileUtil;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
+import java.util.stream.Collectors;
 
 public class ContainerAsFileDataGen extends AbstractDataGen<FileAssetContainer> {
 
+    private class ContentTypeContent {
+        private ContentType contentType;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'contentType' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5041322)